### ISSUE TITLE
Pretty printer for AST

### ISSUE THREAD

**Issue by jml** [jml/graphql-api#67]
`Encoder` is lovely, but it puts everything on one line. For debugging, it would be nice to have a pretty printer.

**Comment by michalrus** [comment]
Yes, please! :pray: Otherwise, how do we write integration tests?
